### DISCUSSION TITLE
specific code platform HAS_UNO_WASM shared file

### DISCUSSION THREAD

**Discussion by Paul2795** [unoplatform/uno#11720] [General]
Hi everyone i'm new to Uno Platform and i'm trying to use Platform-specific code in shared project file

The conditionals that work in the shared file are:
__ANDROID__
__IOS__
HAS_UNO_WINUI

The others that don't work are:
HAS_UNO_WASM
HAS_UNO_SKIA

In particular i would like to use HAS_UNO_WASM in the shared project file.

Someone can help me?
Thanks

**Comment by jeromelaban** [comment]
Thanks for the report. Other users have noticed this, see https://github.com/unoplatform/uno/discussions/11763 for updates.
